### DISCUSSION TITLE
Docs notebook deployment

### DISCUSSION THREAD

**Discussion by bsipocz** [jax-ml/jax#6139] [Q&A]
Hi,

I very much like your approach of having both the md and ipynb files for the documentation in the repo and syncing them. What I wonder about is the reasoning to include the output in the notebooks rather than executing them before deployment, both as a way of testing and as a way to minimizing their size (I have more binaries, figures in my notebooks, so your answer might be that it doesn't matter it here).

Thanks!

**Comment by jakevdp** [comment]
Hi Brigitta! The reason we store outputs in the repo is because the notebooks are designed to be opened via Colab directly from github, and its nice to have the outputs in place for people reading them that way. Otherwise, I think we would probably strip outputs and re-generate them when building the website.

One more thing - some of our notebooks take quite long to run, so we avoid executing them each time we build the docs. That's another potential reason to keep outputs in the source tree.
